seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -2419,6 +2427,10 @@
         "",
         "// Only change code below this line",
         "",
+        "",
+        "// Only change code above this line",
+        "",
+        "timesFive(5);",
```

Comment:
@ChadKreutzer @raisedadead wouldn't this create an error when you initialize the challenge because there is no function defined yet called `timesFive`? I don't think these three lines are necessary. Unless there is another reason why this is here.